content/pages/equipment/equipment.md

```diff
@@ -54,3 +54,7 @@ Check out our [forge lining page](/pages/equipment/forge_lining) for an overview
 [Anvil](https://texasfarriersupply.com/shop/forging-supplies/anvils-and-accessories/anvils/c/980)\
 [Forge](https://mrvolcano.com/collections/the-forge)\
 [Electric inline duct fan](https://www.ebay.com/itm/iPower-4-6-8-Inch-Inline-Duct-Fan-Variable-Speed-Controller-HVAC-Blower-Kit/163765156989)
+
+## Amity
+[1x30 Grinder](https://www.harborfreight.com/1-in-x-30-in-belt-sander-61728.html) 
```

Comment:
This is an idiosyncrasy of Markdown, but can you append a `\` character to the 1x30 grinder line? This makes sure the line break happens correctly (see the rest of the file for examples)